README.md

```diff
@@ -1,8 +1,10 @@
 # FieldSerializer
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/field_serializer`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+Ruby module which allows you to specify a set of fields that you'd like
+serialized in the `#to_h` method. We ([EveryPolitician](http://everypolitician.org/))
+are using this library in our scrapers to make them more declaritive. It allows
```

Comment:
Oops! Fixed in https://github.com/everypolitician/field_serializer/pull/11/commits/be28beb9eff91aabea5872422f837cb62f0e6c95.